src/sdk/namespace_fs.js

```diff
@@ -837,11 +882,11 @@ class NamespaceFS {
     // This is due to MD5 calculation and data buffers
     // Can be finetuned further on if needed and inserting the Semaphore logic inside
     // Instead of wrapping the whole _upload_stream function (q_buffers lives outside of the data scope of the stream)
-    async _upload_stream(params, upload_path, fs_account_config, rpc_client, fs_xattr) {
+    async _upload_stream(params, upload_path, file_path, open_mode, fs_account_config, rpc_client, fs_xattr) {
```

Comment:
The unit tests failure is because you have another call to _upload_stream from upload_multipart() in this file that you need to fix.

```
2022-06-03T20:32:19.6190270Z 
2022-06-03T20:32:19.6196705Z   3319 passing (14m)
2022-06-03T20:32:19.6202078Z   78 pending
2022-06-03T20:32:19.6207777Z   1 failing
2022-06-03T20:32:19.6214785Z 
2022-06-03T20:32:19.6228031Z   1) namespace_fs
2022-06-03T20:32:19.6228432Z        multipart upload
2022-06-03T20:32:19.6228785Z          upload, read, delete a small multipart object:
2022-06-03T20:32:19.6229180Z      TypeError: Cannot convert undefined or null to object
2022-06-03T20:32:19.6229804Z       at NamespaceFS._upload_stream (src/sdk/namespace_fs.js:889:48)
2022-06-03T20:32:19.6230660Z       at /root/node_modules/noobaa-core/src/sdk/namespace_fs.js:973:34
2022-06-03T20:32:19.6231094Z       at Semaphore.surround_count (src/util/semaphore.js:90:90)
2022-06-03T20:32:19.6231516Z       at async NamespaceFS.upload_multipart (src/sdk/namespace_fs.js:971:30)
2022-06-03T20:32:19.6233647Z       at async Context.<anonymous> (src/test/unit_tests/test_namespace_fs.js:255:34)
```

Reply:
fixed